packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js

```diff
@@ -36,14 +206,23 @@ module.exports = new Transformer({
   },
   async transform({asset, options, config}) {
     let exampleCode = [];
+    let exampleImports = [];
     let preReleaseParts = config.version.match(/(alpha)|(beta)|(rc)/);
     let preRelease = preReleaseParts ? preReleaseParts[0] : '';
+    let shouldImportSandpack = false;
     const extractExamples = () => (tree, file) => (
       flatMap(tree, node => {
         if (node.type === 'code') {
           let [meta, ...options] = (node.meta || '').split(' ');
+
+          let shouldUseSandpack = true; // TODO: options.includes('sandpack=true');
```

Comment:
To make this opt-in for examples, I'll switch this to:
```js
let shouldUseSandpack = options.includes('sandpack=true')
```

Reply:
Seems like a good idea, perhaps we should only have the sandpack example for the first example? Or maybe even just a separate playground example in each docs page? Just kinda feels weird to see this when you first land on the docs page, 
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/9661127/160507514-468fa3bc-e2a0-4c5e-a600-e436401dfeaf.png">
feels like the user would only want to tinker around with the more complex examples or after reading through the core documentation: